config/betterquesting/DefaultQuests/Quests/StoringandTransf-AAAAAAAAAAAAAAAAAAAAIQ==/Atlastthefinalca-vrE4rYLuSnmmtAklayd6gw==.json

```diff
@@ -0,0 +1,76 @@
+{
+  "questIDHigh:4": -4705917817748305287,
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": -2921755667014267581,
+      "questIDLow:4": -6876851311040355991
+    }
+  },
+  "questIDLow:4": -6434507911289406845,
+  "properties:10": {
+    "betterquesting:10": {
+      "snd_complete:8": "random.levelup",
+      "taskLogic:8": "AND",
+      "visibility:8": "NORMAL",
+      "isMain:1": 0,
+      "simultaneous:1": 0,
+      "icon:10": {
+        "id:8": "kekztech:kekztech_lapotronicenergyunit_block",
+        "Count:3": 1,
+        "Damage:2": 10,
+        "OreDict:8": ""
+      },
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "§6§lAt last, the final capacitor!",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "This is the last capacitor so we can\u0027t get much higher unfortunately."
+    }
+  },
+  "tasks:9": {
+    "0:10": {
+      "partialMatch:1": 1,
+      "autoConsume:1": 0,
+      "groupDetect:1": 0,
+      "ignoreNBT:1": 1,
+      "index:3": 0,
+      "consume:1": 0,
+      "requiredItems:9": {
+        "0:10": {
+          "id:8": "kekztech:kekztech_lapotronicenergyunit_block",
+          "Count:3": 1,
+          "Damage:2": 10,
+          "OreDict:8": ""
+        }
+      },
+      "taskID:8": "bq_standard:retrieval"
+    }
+  },
+  "rewards:9": {
```

Comment:
UHV lootbags dont exist. its just a placeholder item. They arent planned to be added either.